packages/presentation/lib/di/presentation_injector.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:domain/usecase/check_state_usecase.dart';
+import 'package:domain/usecase/generate_number_usecase.dart';
+import 'package:get_it/get_it.dart';
+import '../screen/main_my_bloc.dart';
+
+void initPresentationInjector() {
+  _initBlocModule();
+}
+
+void _initBlocModule() {
+  GetIt.I.registerFactory<MainBloc>(
+    () => MainBloc(
+      GetIt.I.get<CheckStateUseCase>(),
+      GetIt.I.get<GenerateNumberUseCase>(),
+    ),
+  );
+}
```

Comment:
let's use separating by screen e.g:

void initPresentationInjector() {
  _initMainScreenModule();
}

void _initMainScreenModule() {
  //register here everything relates to main screen (viewmappers, bloc etc.)
  GetIt.I.registerFactory<MainBloc>(
    () => MainBloc(
      GetIt.I.get<CheckStateUseCase>(),
      GetIt.I.get<GenerateNumberUseCase>(),
    ),
  );
}